codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -146,7 +146,9 @@ pub mod evm {
         #[serde(skip_serializing_if = "Option::is_none")]
         #[schemars(
             description = "A flag to indicate if the indexer should rollback to the last known \
-                           valid block on a reorg (default: true)"
+                           valid block on a reorg. This currently incurs a performance hit one \
+                           historical sync and is recommended to to turn this off while \
+                           developing (default: true)"
```

Comment:
```suggestion
                           valid block on a reorg. This currently incurs a performance hit on \
                           historical sync and is recommended to turn this off while \
                           developing (default: true)"
```

Reply:
Thanks JJ, I copied this in locally and ran the generator to output the schema 👍🏼